ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/utils.ts

```diff
@@ -83,3 +86,13 @@ export function formValuesToDeferralRequest(
             return ensureExhaustive(expiryType);
     }
 }
+
+export function formValuesToFalsePositiveRequest(
+    formValues: FalsePositiveValues
+): CreateFalsePositiveVulnerabilityExceptionRequest {
+    return {
+        cves: formValues.cves,
+        comment: formValues.comment,
+        scope: formValues.scope,
+    };
```

Comment:
Is `return formValues` worth considering here, for simplicity?

Reply:
For deferrals we needed a form data -> API type conversion due to the way the expiry field was handled, but that isn't the case for false positives. I wrote this helper for symmetry, but really it isn't doing much.

The only other argument I can think of is that this could remove field data that was added to the object unexpectedly that causes an error when sent over the API. I think chances of that are unlikely though so I can just remove this function call.